pkgs/development/python-modules/pyrodigal/default.nix

```diff
@@ -0,0 +1,33 @@
+{ lib
+, stdenv
+, buildPythonPackage
+, fetchPypi
+, cython
+, archspec
+}:
+
+buildPythonPackage rec {
+  pname = "pyrodigal";
+  version = "2.2.0";
+
+  src = fetchPypi {
+    pname = "pyrodigal";
+    inherit version;
+    hash = "sha256-1RZXeqb4F3ryDRdsB1e7zhc2Ha+1z/wosqMs/xJWwyU=";
+  };
+
+  propagatedBuildInputs = [ archspec ];
+
+  buildInputs = [ cython ];
+
+  doCheck = false;
+
+  pythonImportsCheck = [ "pyrodigal" ];
```

Comment:
Would you mind adding the following, which punches a hole in the sandbox on Darwin that is needed in order to run the import check (at least on my aarch64-darwin machine):
```
sandboxProfile = ''
  # Used by archspec to detect the CPU when importing pyrodigal.
  (allow process-exec (literal "/usr/sbin/sysctl"))
'';
```